.kokoro/downstream-client-library-check.sh

```diff
@@ -86,7 +89,9 @@ if [[ $CLIENT_LIBRARY == "bigtable" ]]; then
   popd
 fi
 
-mvn verify install -B -V -ntp -fae \
--Dmaven.javadoc.skip=true \
--Dgcloud.download.skip=true \
--Denforcer.skip=true
+echo "Modification on the shared dependencies BOM:"
+git diff
+echo
+
+# This reads the JOB_TYPE environmental variable ("test" or "graalvm")
+.kokoro/build.sh
```

Comment:
@Neenu1995 I think it's ok to replace this `mvn verify` with .kokoro/build.sh (with JOB_TYPE: test). What do you think?

Reply:
LGTM.
I don't see any missing use cases.